pkgs/tools/misc/whatsapp-chat-exporter/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, python3Packages, fetchFromGitHub
```

```suggestion
{ lib
, python3Packages
, fetchFromGitHub
```
One parameter per line is preferred.